SlowCheetah/Properties/AssemblyInfo.cs

```diff
@@ -25,6 +25,9 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("6354d859-e629-49fc-b154-fc0ba42d71b0")]
 
+// Allow the test to see the internals of this class
+[assembly: InternalsVisibleTo("SlowCheetah.Tests")]
```

Comment:
I advise against InternalsVisibleTo. By testing only the public API, you are free to refactor all you want without breaking any tests. That tends to mean more refactoring (due to reduced costs and no anxiety about breaking tests), cleaner code, and fewer bugs! :smile:
It also forces you to realize when writing tests that you're missing public API required to do what you want (whether that's provide mocks, assert things, etc.) and you end up building a better API for everyone. Chances are if you test needs to do something, some consumer of your library will too.